exercises/tournament/tests/tournament.rs

```diff
@@ -1,51 +1,57 @@
-use std::fs::File;
-use std::path::Path;
-use std::io::Read;
-
 extern crate tournament;
 
-fn file_equal(output_file: &str, expected_file: &str) {
-    let output = match File::open(&Path::new(output_file)) {
-        Err(e) => panic!("Couldn't open {}: {}", output_file, e),
-        Ok(mut f) => {
-            let mut buf = String::new();
-            match f.read_to_string(&mut buf) {
-                Err(e) => panic!("Couldn't read {}: {}", output_file, e),
-                Ok(_) => buf
-            }
-        }
-    };
-    let expected = match File::open(&Path::new(expected_file)) {
-        Err(e) => panic!("Couldn't open {}: {}", expected_file, e),
-        Ok(mut f) => {
-            let mut buf = String::new();
-            match f.read_to_string(&mut buf) {
-                Err(e) => panic!("Couldn't read {}: {}", expected_file, e),
-                Ok(_) => buf
-            }
-        }
-    };
-    assert_eq!("\n".to_string() + output.as_ref(), "\n".to_string() + expected.as_ref());
-    
-}
-
-
 #[test]
 fn test_good() {
-    assert_eq!(tournament::tally(&Path::new("tests/input1.txt"), &Path::new("tests/output1.txt")).unwrap(), 6);
-    file_equal("tests/output1.txt", "tests/expected1.txt");
+    let input = "Allegoric Alaskians;Blithering Badgers;win\n\
```

Comment:
Yes, with one small modification:

```rust
    let input = "Allegoric Alaskians;Blithering Badgers;win\n".to_string() +
        "Devastating Donkeys;Courageous Californians;draw\n" +
        "Devastating Donkeys;Allegoric Alaskians;win\n" +
        "Courageous Californians;Blithering Badgers;loss\n" +
        "Blithering Badgers;Devastating Donkeys;loss\n" +
        "Allegoric Alaskians;Courageous Californians;win";
```

Else, we would get:

```
tests/tournament.rs:5:17: 5:63 error: binary operation `+` cannot be applied to type `&'static str` [E0369]
tests/tournament.rs:5     let input = "Allegoric Alaskians;Blithering Badgers;win\n" +
                                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Anyway I think your suggestion combines the criteria of "Doesn't require you to realise that the trailing \ strips out the leading whitespace" and "doesn't make the code look weird", so I am going to apply it to all the strings!